src/main/resources/org/jenkinsci/plugins/scriptler/ScriptlerManagement/confirm-remove.js

```diff
@@ -0,0 +1,9 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const removeScriptButtons = document.querySelectorAll(".scriptler-remove-script-form");
+    removeScriptButtons.forEach(button => button.addEventListener("click", (e) => {
```

Comment:
I was reviewing other similar changes and found that some of them use `Behaviour.specify` in cases like this (e.g., jenkinsci/matrix-project-plugin#213). I'm not sure when `Behaviour.specify` should be used and when it shouldn't. Should we use it here to be consistent with Jenkins core and that other PR? Please feel free to disregard this comment if I am wrong about this.

Reply:
I found an explanation here: https://www.jenkins.io/doc/developer/security/csp/#inline-event-handlers